.rubocop.yml

```diff
@@ -31,6 +31,8 @@ Metrics/ParameterLists:
 Style/IfUnlessModifier:
   Enabled: false
 
+Style/Documentation:
+  Enabled: false
+
```

Comment:
we need to have this enabled to make sure we document all classes in the gem

Reply:
right, done!